sdks/python/scripts/run_pylint.sh

```diff
@@ -24,6 +24,13 @@
 # The exit-code of the script indicates success or a failure.
 
 # Check that the script is running in a known directory.
+echo "START"
```

Comment:
my bad will remove this, just wanted to check and try to understand some things....